starterkit_app/test/database/test_isar_database_factory.dart

```diff
@@ -0,0 +1,32 @@
+import 'dart:io';
+
+import 'package:isar/isar.dart';
+import 'package:path/path.dart';
+import 'package:starterkit_app/core/data/database/isar_database_factory.dart';
+
+// Download the latest libraries from https://github.com/isar/isar/releases
```

Comment:
I'll check on this. On the top of my head, I'm thinking of using a custom script that will run every time we run/debug tests via VSCode.

It will:

1. Check if the libraries are existing.
2. Check if their version matches the same version of `Isar` in `pubspec.yaml`.
3. Download or replace the files if they are missing or out of date, for the current platform on which the tests will be running (Windows, macOS, or Linux).

I can integrate this as well as part of the CI for tests. and we can completely remove the library files from git.

Reply:
That would be a great way of automating this. If you think this is out of scope of your masterpriece, feel free to just open an issue to resolve this later.